### ISSUE TITLE
Obsidian Weapons gear bonus (Berserker necklace + Obsidian armour)

### ISSUE THREAD

**Issue by pwatts6060** [LlemonDuck/dps-calculator#70]
Berserker necklace increases the amount of damage the wearer does when wielding an obsidian melee weapon ([Toktz-xil-ak](https://oldschool.runescape.wiki/w/Toktz-xil-ak), [Tzhaar-ket-om](https://oldschool.runescape.wiki/w/Tzhaar-ket-om), [Tzhaar-ket-em](https://oldschool.runescape.wiki/w/Tzhaar-ket-em), or [Toktz-xil-ek](https://oldschool.runescape.wiki/w/Toktz-xil-ek)) by 20%.

When the platebody, platelegs, and helmet are equipped together, all [obsidian weaponry](https://oldschool.runescape.wiki/w/Obsidian_weaponry) will be given a 10% boost in [melee](https://oldschool.runescape.wiki/w/Melee) accuracy and melee strength.

The necklace and armour bonuses are stackable.

Note there is also a berserker necklace (or) with the same effect.

**Comment by kaydiechii** [comment]
correct, this is not accounted for by this plugin in any way, despite obsidian + berserkers being probably one of the most effecient things but varying heavily dedpending on the targets defense, this is the single most desirable usecase for this plugin for me personally.

would like to see it added in

**Comment by DrJam** [comment]
Would also love to see this accounted for
![](https://i.imgur.com/XZg19eN.png)
